drivers/backup/portworx/portworx.go

```diff
@@ -1113,6 +1113,29 @@ func (p *portworx) GetBackupUID(ctx context.Context, backupName string, orgID st
 	return "", fmt.Errorf("backup with name '%s' not found for org '%s'", backupName, orgID)
 }
 
+func (p *portworx) GetBackupName(ctx context.Context, backupUid string, orgID string) (string, error) {
+	var totalBackups int
+	bkpEnumerateReq := &api.BackupEnumerateRequest{OrgId: orgID}
+	bkpEnumerateReq.EnumerateOptions = &api.EnumerateOptions{MaxObjects: uint64(enumerateBatchSize), ObjectIndex: 0}
+	for {
+		enumerateRsp, err := p.EnumerateBackup(ctx, bkpEnumerateReq)
+		log.FailOnError(err, "Failed to enumerate backups for org %s ctx: [%v]\n Backup enumerate request: [%v]", orgID, err, bkpEnumerateReq)
```

Comment:
Can we return the error instead of printing it here?

Reply:
Good catch! I made some changes to the GetBackupUID code without realizing it, so now I need to correct both functions.

Reply:
Modified log message and returned error.